docs/testing/testing.md

```diff
@@ -65,16 +65,8 @@ In this document, the usage of component identifiers like `accessibilityIdentifi
 
 ## Testing for Flutter
 
-* **Flutter app for Android**
-
-For manual testing, the usual way in native Android is testing using Accessibility Scanner. Unfortunatelly that doesn't work with Flutter yet ([see open issue](https://github.com/flutter/flutter/issues/39531)).
-You can use TalkBack, but that's mainly inteded for real users that needs accessibility features. It's good so you know how everything works for user, but it's a bit complex and slow to use for testing.
-
-* **Flutter app for iOS**
-
-For manual testing, you can use XCode Accessibility Inspector (check guide above). It's similar tool as Android's Accesibility Scanner. Advanced tool that users use is VoiceOver which is an equivalent of Android's TalkBack.
-
-For automated testing, main approach is writing widget tests and inspecting that correct semantics is applied.
+Flutter apps can be tested for accessibility in the same way as native apps.
+On Android with the Accessibility Scanner, and on iOS with Accessibility Inspector.
```

Comment:
if we have anything more written about those 2 in native parts, can we please link to them.